circom-types/src/plonk/proof.rs

```diff
@@ -86,6 +87,26 @@ where
     pub curve: String,
 }
 
+impl<P: Pairing + CircomArkworksPairingBridge> PlonkProof<P>
+where
+    P::BaseField: CircomArkworksPrimeFieldBridge,
+    P::ScalarField: CircomArkworksPrimeFieldBridge,
+{
+    /// This function checks whether the group elements are valid.
+    pub fn is_well_constructed(&self) -> Result<(), SerializationError> {
```

Comment:
why do we need this function, does the deserialization not check that anyways?

Reply:
removed